parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -147,6 +147,31 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
+    ORAS_VERSION="1.2.0"
+    CPU_ARCH=$(getCPUArch)
+    
+    # Check if oras is installed and if it is the expected version.
+    if command -v oras &> /dev/null; then
+        INSTALLED_ORAS_VERSION=$(oras version 2>&1 | grep -oP '(?<=Version:\s)\d+\.\d+\.\d+')
+        if [[ "$INSTALLED_ORAS_VERSION" == "$ORAS_VERSION" ]]; then
+            echo "Oras version $ORAS_VERSION is already installed."
+            return
+        else
+            echo "Oras is installed but has the wrong version $INSTALLED_ORAS_VERSION != $ORAS_VERSION. Keeping installed version."
```

Comment:
Question: do we need to uninstall the old version so that the new version can work properly?

Reply:
I actually don't think we really need this check at all. Going to remove